egg/zoo/emcom_as_ssl/data.py

```diff
@@ -78,12 +80,19 @@ def __init__(
             ]
         else:
             transformations = [transforms.Resize(size=(size, size))]
+            
+        if dataset_name = 'imagenet':
+            mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]
+        else:
+            mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]
```

Comment:
Should this be only for cifar? If in the future we want to add other datasets maybe we should leave this optional